app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -26,6 +27,7 @@ describe('awsServerGroupCommandBuilder', function() {
 
     it('initializes to default values, setting usePreferredZone flag to true', function () {
       var command = null;
+      this.settings.providers.aws.defaults.iamRole = '{{application}}IAMRole';
```

Comment:
Ya ... we've already got it in a couple places in `deck` and `clouddriver`. Eventually I could see the defaults being sourced from `clouddriver` rather than duplicated in here (_but would put off doing that for now_).